### ISSUE TITLE
Exception while computing the list of completions

### ISSUE THREAD

**Issue by GuillaumeLeclerc** [xonsh/xonsh#922]
Hello,

when I type: 

```
 echo @($TO<Tab>
```

I get an exception:

```
Exception in thread Thread-119:
Traceback (most recent call last):                                                           
  File "/usr/lib/python3.4/threading.py", line 920, in _bootstrap_inner                      
    self.run()                                                                               
  File "/usr/lib/python3.4/threading.py", line 868, in run                                   
    self._target(*self._args, **self._kwargs)                                                
  File "/usr/local/lib/python3.4/dist-packages/prompt_toolkit/interface.py", line 759, in run
    completions = list(buffer.completer.get_completions(document, complete_event))
  File "/usr/local/lib/python3.4/dist-packages/xonsh/ptk/completer.py", line 35, in get_compls
    self.ctx)
ValueError: need more than 0 values to unpack
```

**Comment by melund** [comment]
I can reproduce this

**Comment by scopatz** [comment]
Seems like this should be a simple fix. Thanks for reporting @GuillaumeLeclerc

**Comment by gforsyth** [comment]
Hey @GuillaumeLeclerc -- can you confirm if this is still a problem on master?  Thanks!

**Comment by GuillaumeLeclerc** [comment]
It does not crash but I don't have any completions at all :/ not sure if it is better

**Comment by gforsyth** [comment]
I believe this was fixed in #1709.  Please reopen if the issue persists!  Thanks!
